client/src/components/messages/Chat.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  ListItem,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+  Chip,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const Chat = ({ profilePictureUrl, body, isOwner }) => {
+  const useStyles = makeStyles((theme) => ({
+    chatContainer: {
+      textAlign: isOwner ? "right" : "left",
+    },
+    messageText: {
+      fontSize: "14px",
```

Comment:
This and other values all already defined in your theme.jsx, you should call them whenever possible.

Reply:
I used this in `<Chip
            className={classes.messageText}
            color={isOwner ? "secondary" : "default"}
            label={body}
          />`

Reply:
I tried [as the Typography in the label props but got on error. 
I also tried Typography as a children of Chip still error.